alembic/versions/2024_05_27_769f8c6595a4_add_sequencing_qc.py

```diff
@@ -0,0 +1,32 @@
+"""Add sequencing QC
+
+Revision ID: 769f8c6595a4
+Revises: ec2db27c06e3
+Create Date: 2024-05-27 10:41:06.062540
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "769f8c6595a4"
+down_revision = "ec2db27c06e3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "case",
+        sa.Column(
+            "sequencing_qc_status",
+            sa.Enum("FAILED", "PASSED", "PENDING", name="sequencingqcstatus"),
```

Comment:
Should this be lower case or upper? The Python models seem to indicate lower case in the values but we might have agreed to use upper case for new enums?

Reply:
UPPER was aggreed upon and should be used for enums regardless